pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -629,13 +651,72 @@ func (b *OFBridge) RetryInterval() time.Duration {
 	return b.retryInterval
 }
 
+func (b *OFBridge) queryTableFeatures() {
+	mpartRequest := &openflow13.MultipartRequest{
+		Header: openflow13.NewOfp13Header(),
+		Type:   openflow13.MultipartType_TableFeatures,
+		Flags:  0,
+	}
+	mpartRequest.Header.Type = openflow13.Type_MultiPartRequest
+	mpartRequest.Header.Length = mpartRequest.Len()
+	tableFeatureCh := make(chan *openflow13.MultipartReply, 10)
```

Comment:
missed that one: you may want to add a comment explaining why 10 is a good choice

Reply:
The comment doesn't explain why 10 is a sensible choice. Is 10 an upper bound on the number of multipart reply messages we expect?

Reply:
I change the number to `20`, which is an observed number of the Multipart Reply messages.